test/asm/string-formatting.err

```diff
@@ -0,0 +1,2 @@
+warning: string-formatting.asm(9): [-Wstring-format]
```

Comment:
Why changing it to a warning?

Reply:
It seemed more appropriate than a fatal error. Similar to "db/dw/dl directive without data in ROM" or "Position is past the end of the string", it's probably an oversight for the dev to correct, not an unrecoverable error.